src/account/FiatExchange.tsx

```diff
@@ -72,21 +83,23 @@ function FiatExchange() {
 
   return (
     <SafeAreaView style={styles.container}>
-      <DrawerTopBar />
+      {!hideDrawerTopNav && <DrawerTopBar testID="FiatExchange/DrawerBar" />}
```

Comment:
I think we'd have to move the top-most `SafeAreaView` out into `FiatExchange` and `WithdrawSpend` in that case; I went with this since it felt a bit simpler.